src/controllers/transfers.js

```diff
@@ -65,12 +66,13 @@ exports.getResource = function * fetch () {
  * @apiVersion 1.0.0
  *
  * @apiDescription Use this to get a signed receipt containing only the id of
- *   transfer and its state.
+ *   transfer and its state. It functions even if the transfer doesn't exist yet.
```

Comment:
I think probably the `/state` endpoint should just return 404 if the transfer does not exist and you didn't pass the `pretend_state` query parameter.

Am I missing something? @emschwartz @sentientwaffle 

I definitely don't like the idea that the ledger will sign something other than the four canonical states of a two-phase commit. Theoretically, signing "nonexistent" could be used to prove that a transfer did *not* happen, so I guess it's useful...